Tests/Tests/CustomAttributeSpec.swift

```diff
@@ -116,6 +124,18 @@ class CustomAttributeSpec: QuickSpec {
                     let att2 = CustomAttribute(withKeyName: "1", withTimeInMilliValue: 13333)
                     expect(att1).toNot(equal(att2))
                 }
+
+                it("will return false when object is not CustomAttribute") {
+                    let att1 = CustomAttribute(withKeyName: "CustomAttribute", withStringValue: "value")
+                    let att2 = "Not CustomAttribute"
+                    expect(att1.isEqual(att2)).to(beFalse())
```

Comment:
use `expect(att1).toNot(equal(att2))` instead of `expect(att1.isEqual(att2)).to(beFalse())`

Reply:
In this case, we want to test the isEqual function inside CustomAttribute to cover the `else {}` part so it will cover all the code.